docs/Text/Parsing/Parser/Combinators.md

```diff
@@ -2,6 +2,28 @@
 
 Combinators for creating parsers.
 
+### Notes:
+A few of the known combinators from Parsec are missing in this module. That
+is because they have already been defined in other libraries.
+
+```purescript
+Text.Parsec.many  = Data.Array.many
+Text.Parsec.many1 = Data.Array.some
+Text.Parsec.(<|>) = Control.Alt.alt (<|>)
+```
+
+Because Strings are not Char Arrays in PureScript `many` and `some` need to
+be used in conjunction with `Data.String.fromCharArray` to achieve "Parsec-like"
+results.
+
+```purescript
+Text.Parsec.many <=> fromCharArray <$> many
```

Comment:
This might be slightly confusing since it doesn't type check. Also, users might use `string` with `many`. How about:

```purescript
Text.Parsec.many (char 'x') <=> fromCharArray <$> many (char 'x')
```

or whatever the right thing is.